packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -22,9 +42,49 @@ declare global {
     interface Chainable {
       /**
        * Start a Wildfly container. This typical command which needs to executed in before method in every TC.
+       * For this case use used unsecured management interface and web console doesn't require the authentication.
```

Comment:
> For this case use used unsecured management interface and web console doesn't require the authentication.

Unsecured management interface is used and web console doesn't require any authentication.

Reply:
updated